src/metabase/api/common.clj

```diff
@@ -123,7 +123,7 @@
 
     (check test1 code1 message1
            test2 code2 message2)"
-  {:style/indent 1, :arglists '([condition [code message] & more] [condition code message & more])}
+  {:style/indent [:form], :arglists '([condition [code message] & more] [condition code message & more])}
```

Comment:
This is a regular function so should get regular function indentation instead of fancy `let`-style indentation, that's just the rules.